tests/search/test_Exchange.py

```diff
@@ -380,10 +380,13 @@ def test_exchange_with_max_duration_constraint(ok_small, op, max_dur, cost):
         route2.append(Node(loc=loc))
     route2.update()
 
-    # Both routes are substantially longer than 5K duration units. So there's
-    # always a max duration violation. Consolidation - either into a single
-    # route, or more into the same route - is typically improving, especially
-    # when the maximum duration violations are significant.
+    # Without maximum duration, route1 has a duration of 5_229 and no time warp
+    # while route2 has a duration of 5_814 and timewarp 2_087, for a net
```

Comment:
That's a bit tricky since the comment mentions what the timewarp would be without a maximum duration. However, in the test we always have a maximum duration so the time warp will be bigger so we can't easily assert this value, unless we make another vehicle type and a new route and use that for this check, while that is not what this test is about. Shall we keep it or do you have a different suggestion?

Reply:
I think this is OK, hadn't considered that the two now interact.